PeerioTechnologies/peerio-desktop
# feat: Windows tray icon

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/15708/desktop-windows-tray-icon
https://app.clubhouse.io/peerio/story/2582/don-t-close-app-on-window-close

Copy: https://github.com/PeerioTechnologies/peerio-icebear/pull/348


#### Testing instructions  

On Windows only:

I.

1. Close the main window.
2. Ensure that Window is hidden.
3. Ensure that a notification saying Peerio is still running appears.
4. Click on the tray icon.
5. Ensure the main window restores.

II.

1. Repeat I, but notice that on step 3 there's no notification anymore (it should appear only once during the lifetime of the app).
2. Quit the app and repeat, making sure there's still no notification.

III.

1. Repeat I, but on step 4 instead right-click.
2. The menu "Show Peerio", "Quit Peerio" should appear.
3. Make sure the items do what they say.

IV.

1. Close the main window.
2. Launch the app again.
3. Make sure there's only once instance of the app and its window restores.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
